src/components/Content/Content.js

```diff
@@ -1,15 +1,214 @@
+    let key = section[0];
+
+    // If Parameters, export both A and B
+    if (key === "parameters") {
+      return (
+        <React.Fragment key="parameters">
+          <button 
+            className={`${section[0]}-a` === activeSection ? "active" : "" } 
+            key={section[0]+"a"} 
+            onClick={() => setActiveSection(section[0]+"-a")}
+          >
+            Parameters A
+          </button>
+          <br />
+          <button 
+            className={`${section[0]}-b` === activeSection ? "active" : "" } 
+            key={section[0]+"b"} 
+            onClick={() => setActiveSection(section[0]+"-b")}
+          >
+            Parameters B
+          </button>
+          <br />
+        </React.Fragment>
+      )
+    } else {
+      return (
+        <React.Fragment key={section[0]}>
+          <button
+            className={section[0] === activeSection ? "active" : "" } 
+            onClick={() => setActiveSection(section[0])}
+          >
+            {section[0]}
+          </button>
+          <br />
+        </React.Fragment>
+      );
+    }
+  });
+};
+
+const formatQuestion = (question, variables, setVariables) => {
+  if (question == null) {
+    return;
+  }
+  const key = question[0];
+  const value = question[1];
+
+  let required = false;
+  if (value?.required === true) {
+    required = true;
+  }
+
+  switch (value?.type) {
+    case "float": // fall through
+    case "integer":
+      return (
+        <InputGroup key={key}>
+          <Label for={key}>{value.label}</Label>
+          <Input 
+            type="number" 
+            required={required}
+            name={key}
+            id={key}
+            defaultValue={variables[key]}
+            onChange={input => setVariables({...variables, [key]: input.currentTarget.value})}
+          />
+          {value.unit && (
+            <InputGroupAddon addonType="append">
+              <InputGroupText>{value.unit}</InputGroupText>
+            </InputGroupAddon>
+          )}
+        </InputGroup>
+      );
+    case "text":
+      return (
+        <InputGroup key={key}>
+          <Label for={key}>{value.label}</Label>
+          <Input 
+            type="text" 
+            required={required}
+            name={key}
+            id={key}
+            defaultValue={variables[key]}
+            onChange={input => setVariables({...variables, [key]: input.currentTarget.value})}
+          />
+          {value.unit && (
+            <InputGroupAddon addonType="append">
+              <InputGroupText>{value.unit}</InputGroupText>
+            </InputGroupAddon>
+          )}
+        </InputGroup>
+      );
+    case "gap":
+      return <div key={key} className="spacer">&nbsp;</div>;
+    default:
+      return;
+  }
+};
+
 function Content() {
+  const [activeSection, setActiveSection] = useState("introduction");
+  const [activeSubsection, setActiveSubsection] = useState(null);
+  const [variables, setVariables] = useState({});
```

Comment:
As mentioned before, creating a `setVariable` function would help reduce what you need to pass down to props and increase performance.

```suggestion
  const [variables, setVariables] = useState({});
  const handleVariableChange = (event) => {
    setVariables({
      ...variables,
      [event.target.name]: event.target.value,
    });
  };
```